src/index.html

```diff
@@ -1,13 +1,52 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a href="#" class="card__image-link">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac image"
+          class="card__image"
+          >
+      </a>
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="card__reviews-stars stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews-count">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+
+      <a
+          href="#"
+          class="card__button"
+          data-qa="hover">Buy
+      </a>
```

Comment:
https://github.com/mate-academy/layout_product-cards/pull/3712#discussion_r1365093521 - not fully fixed, see an example of good formatting and update this part of the code.